src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -119,8 +119,13 @@ class OrchestrationSessionsController(private val visitSchedulerService: VisitSc
     )
     @NullableNotEmpty(message = "An empty visitors list is not allowed")
     visitors: List<Long>? = null,
+    @RequestParam(value = "withAppointmentsCheck", required = false)
+    @Parameter(
+      description = "Defaults to true if not passed. If true, will not return visit times that clash with higher priority legal or medical appointments.",
+    )
+    withAppointmentsCheck: Boolean? = true,
```

Comment:
have kept this non mandatory for now as am not sure if PVB needs this flag. Will check with Simon / Omar.

Reply:
Checked with Simon who said he is happy with this approach